### ISSUE TITLE
Darken the red used for vertex outlines and proposed connections

### ISSUE THREAD

**Issue by arouinfar** [phetsims/circuit-construction-kit-common#1226]
For #1178 

The red used for unconnected vertex outlines and proposed connections has insufficient contrast against the sim's background.

<img width="189" height="99" alt="Image" src="https://github.com/user-attachments/assets/0d5c4752-c323-461b-bfc8-89a96e592ff7" />

@samreid please change both of these reds to `#D10000` or `rgb( 209, 0, 0 )` (whichever color system you prefer).

**Comment by samreid** [comment]
OK I changed the red, @arouinfar can you please test and close if ok?
